OpenOversight/tests/routes/test_user_api.py

```diff
@@ -155,12 +155,15 @@ def test_admin_cannot_delete_other_admin(client, session):
 
 def test_admin_can_disable_user(client, session):
     with current_app.test_request_context():
-        login_admin(client)
+        _, current_user = login_admin(client)
 
         # just need to make sure to not select the admin user
-        user = User.query.filter_by(is_administrator=False).first()
+        user = User.query.filter_by(
+            is_administrator=False, disabled_at=None, disabled_by=None
+        ).first()
 
-        assert not user.is_disabled
+        assert user.disabled_at is None
+        assert user.disabled_by is None
```

Comment:
These are the same statements, but they feel more intuitive written like this. I'd be happy to switch them back tho.